src/styles/variables.scss

```
@@ -4,3 +4,5 @@ $header-height: 66px;
 $menubar-height: 24px;
 
 $detail-view-splitter-cell-z-index: 2;
+
+$character-item-size: 215px;
```

variablesに移動、良さそう。

なにかわからなくなりそうなのでとりあえずこんな感じのコメント書いとくのどうでしょ。
```suggestion
$character-item-size: 215px; // キャラクター並び替えダイアログなどでの１キャラ表示要素の横幅
```

---

そうですね、そのままsuggestを利用させてもらいます...!